### DISCUSSION TITLE
Face recognition with tag keywords

### DISCUSSION THREAD

**Discussion by davidetoile** [photoprism/photoprism#2939] [Q&A]
Hi,
I have already tagged all person faces in my photos using single keywords by hand, like "John DOE". Is there a way to use this information to populate the person database? It is very frustrating as it does not use my keywords which the system can read!
Thanks

**Comment by graciousgrey** [comment]
We have a few related feature requests to import face metadata written by other tools:

- #747 
- #554
- #1570

I'm not sure whether creating people from keywords would work well, since keywords are used for many other things as well. Even if we have an algorithm that recognizes names in keywords, we can't know if a photo has that name as a keyword because the person is in the picture or because the person is the artist, for example.
